src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlByteArrayMethodTranslator.cs

```diff
@@ -33,43 +29,56 @@ public NpgsqlByteArrayMethodTranslator([NotNull] ISqlExpressionFactory sqlExpres
             Check.NotNull(method, nameof(method));
             Check.NotNull(arguments, nameof(arguments));
 
-            // Note: we only translate if the array argument is a column mapped to bytea. There are various other
-            // cases (e.g. Where(b => new byte[] { 1, 2, 3 }.Contains(b.SomeByte))) where we prefer to translate via
-            // regular PostgreSQL array logic.
-            if (method.IsGenericMethod
-                && method.GetGenericMethodDefinition().Equals(ContainsMethod)
-                && arguments[0].TypeMapping is NpgsqlByteArrayTypeMapping typeMapping)
+            if (method.IsGenericMethod && arguments[0].TypeMapping is NpgsqlByteArrayTypeMapping typeMapping)
             {
-                var source = arguments[0];
+                // Note: we only translate if the array argument is a column mapped to bytea. There are various other
+                // cases (e.g. Where(b => new byte[] { 1, 2, 3 }.Contains(b.SomeByte))) where we prefer to translate via
+                // regular PostgreSQL array logic.
+                if (method.GetGenericMethodDefinition().Equals(EnumerableMethods.Contains))
```

Comment:
Out of curiosity, is there a reason to use `Equals` instead of the `==` operator?

Reply:
I don't think so - I sometimes use equality, but the EF team prefer Equals. I think there was an issue on some platform/TFM at some point, I really don't know anymore...